src/push/plugin.ts

```diff
@@ -61,21 +68,43 @@ export function MultiAssetPlugin(callback: PluginCallback): esbuild.Plugin {
     return false;
   };
 
+  // If we're importing the @elastic/synthetics package
+  // directly from source instead of using the fully
+  // qualified name, we must skip it too. That's just
+  // so it doesn't get bundled on tests or when we locally
+  // refer to the package itself.
+  const isLocalSynthetics = (entryPath: string) => {
+    return entryPath.startsWith(sourceSyntheticsPath);
+  };
+
+  // When importing the local synthetics module directly
+  // it may import its own local dependencies, so we must
+  // make sure those will be resolved using Node's resolution
+  // algorithm, as they're still "node_modules" that we must bundle
+  const isLocalSyntheticsModule = (str: string) => {
```

Comment:
Why not do str.includes`@elastic/synthetics`, it would work with source node modules and synthetics itself? We would get absolute path all the time, so this should be good enough?

Reply:
I might have misunderstood your comment, but yes we do get the absolute path (when importing the `.js`, not always). That's the exact reason we can't use the `includes` you suggested because when importing the local file the alias will not be the name we're passing in.

Is this what you meant? I thought I might have misunderstood because even if what you suggest happens always that means we can never use the suggested `includes`.